backend-models/service-tests/sectionedservice-test.txt

```diff
@@ -0,0 +1,551 @@
+sectionedservice.php
+
+----------------------------------
+getGroupsTest() line 100:
+----------------------------------
+
+*********
+ Pre-req
+*********
+
+(checklogin) IS TRUE
+
+//// PRE-REQ Values /////
+
+send
+{
+$opt = "GRP"
+$courseid = "1"
+$coursevers = "45656"
+}
+
+/////////////////////////
+
+
+*****************
+ Compare results
+*****************
+
+If $debug is empty then successfully gathered all groups
+
+*****************
+ Service output
```

Comment:
the output that is to be recorded are the values in this array 
![image](https://user-images.githubusercontent.com/102600391/235639630-37485d2e-4753-4c8f-83aa-b3761d3c86eb.png)
![image](https://user-images.githubusercontent.com/102600391/235639728-20b2f1cf-f3cf-4d84-a920-e6661a3caec9.png)